src/Utilities/ProtocolHelpers.hpp

```diff
@@ -20,21 +31,22 @@
  * instead. Set the `SPECTRE_ALWAYS_CHECK_PROTOCOL_CONFORMANCE` CMake option to
  * always enable rigorous protocol conformance checks.
  *
- * \see \ref protocols
+ * \see Documentation on \ref protocols
  */
 #ifdef SPECTRE_ALWAYS_CHECK_PROTOCOL_CONFORMANCE
-template <typename ConformingType, typename Protocol>
+template <typename ConformingType, template <class> class Protocol>
```

Comment:
the template template can be `template <typename> class` (both don't have to be class), though I don't think it has any functional difference, so I'll leave it up to you for whether you want to change it.

Reply:
I chose `class` for both to avoid confusion why one is `class` and the other is `typename` 🤷‍♂️